src/crimson/os/seastore/transaction_manager.cc

```diff
@@ -329,10 +329,22 @@ TransactionManager::update_lba_mappings(
         // for rewritten extents, last_committed_crc should have been set
         // because the crc of the original extent may be reused.
         // also see rewrite_logical_extent()
-        if (!extent->get_last_committed_crc()) {
-          extent->set_last_committed_crc(extent->calc_crc32c());
-        }
-        assert(extent->calc_crc32c() == extent->get_last_committed_crc());
+	if (!extent->get_last_committed_crc()) {
+	  if (!extent->get_paddr().is_absolute() ||
```

Comment:
In the update-mapping path (applies to all the other places), looks unnecessary after `is_fake()` is considerred.